script/test_todo.py

```diff
@@ -46,6 +46,31 @@ def verify_all_completed_task_are_deleted(driver):
     assert elem.get_attribute("class") != "completed"
 
 
+def click_tab_active(driver):
+  tab_active = driver.find_element_by_xpath("/html/body/section/div/footer/ul/li[2]/a")
+  tab_active.click()
+
+
+def click_tab_completed(driver):
+  tab_completed = driver.find_element_by_xpath("/html/body/section/div/footer/ul/li[3]/a")
+  tab_completed.click()
+
+
+def verify_tab_all(driver):
```

Comment:
Also here create one function `verify_tab` which will verify tab based on the parameter